File: ios/TimelessApp/TimelessApp/DesignSystem/Tokens/Gradients.swift
import SwiftUI

/// Timeless Design System Gradients
extension LinearGradient {
    /// Primary gradient - 科技紫漸層
    static let primary = LinearGradient(
        colors: [.primary500, .accent400],
        startPoint: .topLeading,
        endPoint: .bottomTrailing
    )

    /// Neon glow gradient
    static let neonGlow = LinearGradient(
        colors: [.primary400, .accent400, .secondary400],
        startPoint: .leading,
        endPoint: .trailing
    )

    /// Card background gradient
    static let cardBackground = LinearGradient(
        colors: [.backgroundCard, .backgroundElevated],
        startPoint: .top,
        endPoint: .bottom
    )

    /// Button gradient
    static let button = LinearGradient(
        colors: [.primary500, .primary600],
        startPoint: .top,
        endPoint: .bottom
    )
}

File: android/app/src/main/kotlin/com/timeless/app/designsystem/tokens/Gradients.kt
package com.timeless.app.designsystem.tokens

import androidx.compose.ui.graphics.Brush

/**
 * Timeless Design System Gradients
 */

// Primary gradient - 科技紫漸層
val PrimaryGradient = Brush.linearGradient(
    colors = listOf(Primary500, Accent400)
)

// Neon glow gradient
val NeonGlowGradient = Brush.horizontalGradient(
    colors = listOf(Primary400, Accent400, Secondary400)
)

// Card background gradient
val CardBackgroundGradient = Brush.verticalGradient(
    colors = listOf(BackgroundCard, BackgroundElevated)
)

// Button gradient
val ButtonGradient = Brush.verticalGradient(
    colors = listOf(Primary500, Primary600)
)
